bundle/config/variable/variable.go

```diff
@@ -4,10 +4,12 @@ import (
 	"fmt"
 )
 
+type VariableValue = any
```

Comment:
Please include a comment explaining why this is any and why this is its own type.